centreon/src/Core/HostTemplate/Application/Repository/ReadHostTemplateRepositoryInterface.php

```diff
@@ -173,4 +173,12 @@ public function findNamesByIds(array $templateIds): array;
      * @return HostTemplate[]
      */
     public function findAll(): array;
+
+    /**
+     * Find direct parent templates of a host id.
+     *
+     * @param int $hostId
+     * @return array<int, int>
+     */
+    public function findHostTemplatesByHostId(int $hostId): array;
```

Comment:
You repeat the context, you can simplify by : 
```suggestion
    public function findByHostId(int $hostId): array;
```

Reply:
good !